.github/workflows/macos-executable-release.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         APPLE_TEAM_ID: ${{ secrets.APPLE_TEAM_ID}}
         FILE_PATH: dist/macos/${{ github.event.inputs.formula }}-v${{ needs.get-tag.outputs.TAG_NAME }}.pkg
     - name: Upload binaries to release
+      if: always()
```

Comment:
Any way to figure out if it's a specific keytar issue? I mean, it adds a blanket approval to upload the binary even if the last step failed for some other reason, and we'd not really know about it.

Reply:
So the above steps include packaging -> notarisation.  If packaging fails upload binaries to release will also fail since its will not find the package to release. for notarisation it can fail for different reasons, but we can still upload to binary, are you suggesting we only upload if notarisation returns the issue is for keytar?  for that we would need to parse the notarisation logs, and ensure if the issue is only for keytar. But I don't understand how it will help the end user, because they will not be aware of the reason of failure in notarisation.